modules/govuk_gor/manifests/init.pp

```diff
@@ -16,11 +16,19 @@
 # [*version*]
 #   Which version of the Go package to install; defaults to latest
 #
+# [*binary_path*]
+#   Explicitly set the location of the Gor binary.
+#
+# [*envvars*]
+#   Set any environment variables that will be loaded into the Gor service.
+#
 class govuk_gor(
   $apt_mirror_hostname = '',
   $args = {},
   $enable = false,
   $version = '0.9.4~ppa1',
+  $binary_path = undef,
```

Comment:
One thing I'm a little unsure of is what this will `undef` will do when it's passed through to the gor module, will it override the `/usr/bin/gor` from that module?